ledger/participant-integration-api/src/main/scala/platform/apiserver/ApiService.scala

```diff
@@ -7,7 +7,7 @@ import com.daml.ports.Port
 
 import scala.concurrent.Future
 
-trait ApiServer {
+trait ApiService {
```

Comment:
I think this rename is solid. Although I am even afraid to say it out loud: the packagename apiserver is now also asks for a rename (definitely NOT in this PR). WDYT?

Reply:
TODO fine

Reply:
Added a TODO